final_task/exceptions/exception_handler.py

```diff
@@ -0,0 +1,8 @@
+import feedparser
+
+
+class ExceptionHandler(Exception):
+    """Handles all exception that I provided"""
+
+    def __init__(self, exc):
+        self.exc = exc

```

Comment:
A question:

Why do you need this class? If you want to raise your custom exceptions you can just implement them.

If you want to be able to handle all exceptions using only one class just use `except Exception` (although it is a bad practice)